content/src/Blip.Api.Template/Helpers/BlipHelper.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Lime.Protocol;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Blip.Api.Template.Helpers
+{
+    public class BlipHelper
+    {
+        public static string GetBotAuthorization(string identifier, string accessKey)
```

Comment:
useless method as the `MySettings` class already has the bot's Authorization header